tests/cpp-tests/Classes/NewAudioEngineTest/NewAudioEngineTest.cpp

```diff
@@ -189,7 +190,7 @@ AudioEngineTestDemo::AudioEngineTestDemo()
 void AudioEngineTestDemo::onExit()
 {
     *_isDestroyed = true;
-    AudioEngine::stopAll();
+    AudioEngine::uncacheAll();
```

Comment:
After this line was changed to `uncacheAll`, it will trigger a bug http://punchbox.info:3000/issues/26038.
It only crash on win32 platform. I will try to fix it.